presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestingDatabase.java

```diff
@@ -96,16 +97,16 @@ public JdbcClient getJdbcClient()
 
     public JdbcSplit getSplit(String schemaName, String tableName)
     {
-        JdbcTableHandle jdbcTableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
-        JdbcTableLayoutHandle jdbcLayoutHandle = new JdbcTableLayoutHandle(jdbcTableHandle, TupleDomain.all());
+        Optional<JdbcTableHandle> jdbcTableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
+        JdbcTableLayoutHandle jdbcLayoutHandle = new JdbcTableLayoutHandle(jdbcTableHandle.orElse(null), TupleDomain.all());
```

Comment:
`JdbcTableLayoutHandle` does not accept a null table handle. Probably want to do
```java
SchemaTableName table = new SchemaTableName(schemaName, tableName)
JdbcTableHandle jdbcTableHandle = jdbcClient.getTableHandle(table);
        .orElseThrow(() -> new IllegalArgumentException("table not found: " + table)));
JdbcTableLayoutHandle jdbcLayoutHandle = new JdbcTableLayoutHandle(jdbcTableHandle, TupleDomain.all());